### PR TITLE
Remove music

### PR DESCRIPTION
**Pull request by yazninja**

### CODE REVIEW

**Review by coredev-uk — APPROVED** [review]

File: src/commands/general/ping.ts [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
-        .addBooleanOption(option => option
-            .setName('show')
-            .setDescription('Show to everyone')
-            .setRequired(false)
-        ),
+        .addBooleanOption((option) => option.setName('show').setDescription('Show to everyone').setRequired(false)),
     category: 'General',
-    async execute(interaction : ChatInputCommandInteraction) {
+    async execute(interaction: ChatInputCommandInteraction) {
         let show = interaction.options.getBoolean('show') || false;
```
Comment (Amaru8) [inline src/commands/general/ping.ts:9]: Hey! Can you remove the show variable entirely and just directly set ephemeral to true please. Thanks, nice pull! 😊

**Review by Amaru8 — CHANGES_REQUESTED** [review]
Hey! Can you remove the show variable entirely and just directly set ephemeral to true please. Thanks, nice pull! 😊

File: src/commands/general/ping.ts (hunk 1 above)
Comment (yazninja) [reply src/commands/general/ping.ts:9]: so this means that it doesn;t show the ping to everyone?

Comment (Amaru8) [reply src/commands/general/ping.ts:9]: I don't really think that showing the ping is useful for anyone other then the person specifically requesting the data.

**Pull request closed without merging**